googleapiclient/errors.py

```diff
@@ -44,9 +44,9 @@ def _get_reason(self):
     """Calculate the reason for the error from the response content."""
     reason = self.resp.reason
     try:
-      data = json.loads(self.content)
+      data = json.loads(self.content.decode('utf-8'))
```

Comment:
http response should bytes always. It should have `.decode()` on both of PY2 and PY3.
AttributeError is just a bug. Catching it is bad idea.